src/app/TypeormStore/TypeormStore.test.ts

```diff
@@ -195,15 +237,26 @@ class Test {
     this.express.get("/views", (req, res) => {
       const session = nullthrows(req.session);
 
-      res.json(session.views || 0);
+      res.json((session as any).views || 0);
```

Comment:
I'm having some issues with TS... I had to add "as any" in many places... will remove as soon as I understand how.